### PR TITLE
Add conformance endpoint.

### PR DESCRIPTION
**Pull request by Jeffrey-Vervoort-KNMI**

### CODE REVIEW

**Review by Teddy-1000 — APPROVED** [review]

File: api/metadata_endpoints.py [hunk 1]
```diff
@@ -36,12 +36,29 @@ def get_landing_page(request):
...
+            Link(href=f"{request.url}conformance", rel="conformance", title="Conformance Declaration in JSON"),
             Link(href=f"{request.url}collections", rel="data", title="Collections metadata in JSON"),
         ],
     )
 
 
+def get_conformance() -> ConformanceModel:
+    return ConformanceModel(
+        conformsTo=[
+            "https://www.opengis.net/spec/ogcapi-common-1/1.0/conf/core",  # A2 - required
```
Comment (lukas-phaf) [inline api/metadata_endpoints.py:48]: In the OGC EDR example, the links are slightly different:
https://docs.ogc.org/is/19-072/19-072.html#_d5e2189c-9836-4af5-9d91-69b02cd1eab1

The first one for example is:
"http://www.opengis.net/spec/ogcapi-common-1/1.0/req/core"
Do we now what the differences is, and why did we pick the ones with `conf`?

Also, what about the others in that example, do we conform to those (like the `landing-page` one)?

Also, there seems to be a `json` and `html` in both `common` and `edr`. Why did we pick one and not the other?

File: api/metadata_endpoints.py [hunk 2]
```diff
@@ -36,12 +36,29 @@ def get_landing_page(request):
...
+            "https://www.opengis.net/spec/ogcapi-edr-1/1.1/req/collections",  # A3 - required
+            "https://www.opengis.net/spec/ogcapi-edr-1/1.1/req/queries",  # A4
+            "https://www.opengis.net/spec/ogcapi-edr-1/1.1/conf/core",  # B2
+            "https://www.opengis.net/spec/ogcapi-edr-1/1.1/conf/collections",  # B3
+            "https://www.opengis.net/spec/ogcapi-edr-1/1.1/conf/json",  # B4
+            "https://www.opengis.net/spec/ogcapi-edr-1/1.1/conf/geojson",  # B5
+            "https://www.opengis.net/spec/ogcapi-edr-1/1.1/conf/covjson",  # B7
+            "https://www.opengis.net/spec/ogcapi-edr-1/1.1/conf/html",  # B8
+            "https://www.opengis.net/spec/ogcapi-edr-1/1.1/conf/oas30",  # B9
```
Comment (lukas-phaf) [inline api/metadata_endpoints.py:57]: Maybe my question is really: Why did we list these, and did we check we actually conform?

File: api/metadata_endpoints.py (hunk 1 above)
Comment (Jeffrey-Vervoort-KNMI) [reply api/metadata_endpoints.py:48]: They fixed the URLs. The example initially contained broken links and URLs with conf. I tested the URLs and now all the URLs in the example of the spec work. Will look into what the difference between `conf` and `req` is.

Comment (Jeffrey-Vervoort-KNMI) [reply api/metadata_endpoints.py:48]: At the moment the examples for 1.1 are back at `conf` again: https://docs.ogc.org/is/19-086r6/19-086r6.html#_19183664-f3c9-4df4-b6e8-4afa9c54ac08

Comment (Jeffrey-Vervoort-KNMI) [reply api/metadata_endpoints.py:48]: Requirements belong to a Conformance Class. Each Requirements Class defines a functional requirement. Conformance classes are advertised through the `/conformance` path.

**Review by lukas-phaf — APPROVED** [review]

**Pull request merged**